js/src/tooltip.js

```diff
@@ -614,10 +621,15 @@ const Tooltip = (() => {
     }
 
     _getConfig(config) {
-      config = $.extend(
+      if (typeof config !== 'undefined' &&
+        typeof config.container === 'object' && config.container.jquery) {
```

Comment:
what is it `config.container.jquery` ?

Reply:
Checks if `config.container` is a jquery object. In that case i'll get the dom element via `[0]` .

Reply:
that's right, I forgot about that property :smile: